ebml/EbmlId.h

```diff
@@ -55,11 +55,10 @@ namespace libebml {
 class EBML_DLL_API EbmlId {
   public:
     EbmlId(const binary aValue[4], const unsigned int aLength)
-      :Length(aLength)
+      :Value(0)
```

Comment:
Not a fan of that. It gives the wrong impression that's the default value for an ID.